ocfweb/api/hours.py

```diff
@@ -51,6 +51,14 @@ def _get_hours():
     return all_shifts
 
 
+def _merge_shifts(first, second, staffer=True):
+    return Hour(
+        open=min(first.open, second.open),
+        close=max(first.close, second.close),
+        staffer=first.staffer if staffer else None,
```

Comment:
I don't really like having this pattern of having a staffer associated with each `Hour` object, even if it doesn't actually mean anything. It's not a big deal for this pull request at least, but it seems like a pattern that should be fixed in the future sometime.

Reply:
The idea is that in the near future we can use this to run reports of sorts on whether or not staffers actually showed up to their shifts. If we integrate labstats into ocfweb, we can then have everything run nicely and concisely.

Reply:
That's a good point with labstats being merged, but maybe it would be possible to just not even need to put a staffer here and have it be optional or something?